### PR TITLE
Add schnetpack

### PR DESCRIPTION
**Pull request by jan-janssen**
@conda-forge/help-python 

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/schnetpack/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - pytorch-cpu >=0.4.1
+    - numpy
+    - ase >=3.18
+    - h5py
+    - tensorboardX
```
Comment (chrisburr) [inline recipes/schnetpack/meta.yaml]: Conda packages are always lowercase:
```suggestion
    - tensorboardx
```

File: recipes/schnetpack/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - tqdm
+    - pyyaml
+    - pytest-console-scripts
```
Comment (bgruening) [inline recipes/schnetpack/meta.yaml]: is pytest really a runtime dependency?

Comment (jan-janssen) [reply recipes/schnetpack/meta.yaml]: It is just used for testing I guess, but unfortunately it is listed in `install_requires`

Comment (jan-janssen) [reply recipes/schnetpack/meta.yaml]: I opened a PR upstream https://github.com/atomistic-machine-learning/schnetpack/pull/164

**Comment by jan-janssen** [comment]
@bgruening The pull request was merged and I am now removing the pytest package from the requirements https://github.com/conda-forge/schnetpack-feedstock/pull/1

**Comment by bgruening** [comment]
Awesome! I love it when conda-forge contributions help cleaning up upstream! win-win!

**Pull request merged**